### PR TITLE
feat(playground): toggle source code visibility

### PR DESCRIPTION
**Pull request by sean-perkins**
Introduces the ability to toggle the visibility of the code block.

<video src="https://user-images.githubusercontent.com/13732623/153088863-fd67519e-9347-4563-ae58-691ad34592c7.mp4"></video>

Split from #2185 continuation of #2189

### CODE REVIEW

File: src/components/global/Playground/index.tsx [hunk 1]
```diff
@@ -9,6 +11,7 @@ enum Mode {
 
 export default function Playground() {
   const [mode, setMode] = useState(Mode.iOS);
+  const [expanded, setExpanded] = useState(false);
```
Comment (liamdebeasi) [inline src/components/global/Playground/index.tsx:14]: Can we make this variable more descriptive? `codeExpanded` or something

File: src/components/global/Playground/index.tsx [hunk 2]
```diff
@@ -38,7 +41,23 @@ export default function Playground() {
             </button>
           </div>
           <div className="playground__control-group playground__control-group--end">
-            {/* TODO FW-737: Toggle/Collapse Button */}
+            <button
+              className="playground__icon-button"
+              aria-label={expanded ? 'Hide source code' : 'Show full source'}
+              onClick={() => setExpanded(!expanded)}
+            >
+              <svg
```
Comment (liamdebeasi) [inline src/components/global/Playground/index.tsx:49]: The rendered SVG looks very small compared to what's in the Figma design

Comment (sean-perkins) [reply src/components/global/Playground/index.tsx:49]: Updated 👍  was having an issue with that asset originally, but looks like Ben fixed it.

**Review by liamdebeasi — APPROVED** [review]
Good to go other than the comment update

**Pull request merged**